### PR TITLE
#4 | Added countLogger

### PR DESCRIPTION
**Pull request by ferdyh**

### CODE REVIEW

File: src/main/scala/nl/getthere/dataunit/spark/utils/run/CountLogger.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      .range(1)
+      .select(
+        lit(Databricks.getRunId.getOrElse(0)).as("run_id"),
+        lit(database).as("database"),
+        lit(table).as("table"),
+        lit(count).as("count")
+      )
+      .write
+      .format("parquet")
+      .mode(SaveMode.Append)
```
Comment (frankivo) [inline src/main/scala/nl/getthere/dataunit/spark/utils/run/CountLogger.scala:23]: prefer "append" to save the import

File: src/test/scala/nl/getthere/dataunit/spark/utils/run/CountLoggerTest.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package nl.getthere.dataunit.spark.utils.run
+
+import nl.getthere.dataunit.spark.utils.Databricks
+import org.scalatest.funsuite.AnyFunSuite
+
+import scala.reflect.io.Directory
+
+class CountLoggerTest extends AnyFunSuite {
+
+  Directory(new java.io.File("./spark-warehouse")).deleteRecursively()
```
Comment (frankivo) [inline src/test/scala/nl/getthere/dataunit/spark/utils/run/CountLoggerTest.scala:10]: should we do this at the end of the tests too?

**Review by frankivo — CHANGES_REQUESTED** [review]

File: src/main/scala/nl/getthere/dataunit/spark/utils/run/CountLogger.scala (hunk 1 above)
Comment (ferdyh) [reply src/main/scala/nl/getthere/dataunit/spark/utils/run/CountLogger.scala:23]: Wrote it wrong few times... This atleast gets picked up by the compiler; Prefer it in notebooks aswell, but maybe in library this is better..? IDE helps you anyways.

File: src/test/scala/nl/getthere/dataunit/spark/utils/run/CountLoggerTest.scala (hunk 2 above)
Comment (ferdyh) [reply src/test/scala/nl/getthere/dataunit/spark/utils/run/CountLoggerTest.scala:10]: I'm not really sure; Before you're atleast certain its all right for these tests. Ofcourse cleaning up afterwards is neat. Probably need create an issue to fix pre and post test stuff, and maybe a test data model?

File: src/main/scala/nl/getthere/dataunit/spark/utils/run/CountLogger.scala (hunk 1 above)
Comment (frankivo) [reply src/main/scala/nl/getthere/dataunit/spark/utils/run/CountLogger.scala:23]: ok :)

File: src/test/scala/nl/getthere/dataunit/spark/utils/run/CountLoggerTest.scala (hunk 2 above)
Comment (frankivo) [reply src/test/scala/nl/getthere/dataunit/spark/utils/run/CountLoggerTest.scala:10]: Yep, let's do that later

**Review by frankivo — APPROVED** [review]

**Pull request merged**